resources/apecoin/index.ts

```diff
@@ -23,6 +24,7 @@ const config: IDAOConfig = {
     ORDERSTAT: 'karmaScore',
   },
   DAO_DELEGATE_CONTRACT: undefined,
```

Comment:
Delegate contract is the snapshot contract, this shouldn't be undefined?

Reply:
It's undefined because when we have a DAO that uses snapshot, it will use snapshot contract address. So we let "undefined" to avoid problems of the code looking a wrong contract address instead of snapshot.